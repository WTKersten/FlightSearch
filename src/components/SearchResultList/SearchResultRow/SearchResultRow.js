import React, { PureComponent } from "react";
import PropTypes from "prop-types";
import "./SearchResultRow.scss";

class SearchResultRow extends PureComponent {
  render() {
    return (
      <div className="component-search-result-row">
        <span className="from">{this.props.from}</span>
        <span className="to">{this.props.to}</span>
        <span className="price">{this.props.price}</span>
      </div>
    );
  }
}
SearchResultRow.propTypes = {
  id: PropTypes.number,
  from: PropTypes.string,
  to: PropTypes.string,
  price: PropTypes.string  
};
export default SearchResultRow;
