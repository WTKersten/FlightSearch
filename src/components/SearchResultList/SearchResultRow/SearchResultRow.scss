@import '../../../styles/variables.scss';

.component-search-result-row {
  padding: 20px 10px;
  border: 1px solid $color-grey;

  .from::after {
    content: ' - '
  }
  .price {
    color: $color-green;
    float: right;
  }
}