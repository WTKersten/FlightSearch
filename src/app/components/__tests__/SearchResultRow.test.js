import React from 'react';
import { shallow } from 'enzyme';
import SearchResultRow from '../SearchResultRow/SearchResultRow';

describe('SearchResultRow', () => {
  it('renders correctly', () => {
    const component = shallow(<SearchResultRow />);
    expect(component).toMatchSnapshot();
  });
});