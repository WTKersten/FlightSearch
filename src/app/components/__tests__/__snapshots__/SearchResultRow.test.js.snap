// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultRow renders correctly 1`] = `
<div
  className="component-search-result-row"
>
  <span
    className="from"
  />
  <span
    className="to"
  />
  <span
    className="price"
  />
</div>
`;
